jenkins/scripts/perf/local/submit.py

```
@@ -0,0 +1,530 @@
+    draft_launch_lines = remove_whitespace_lines(draft_launch_lines)
+    draft_launch_content = "\n".join(draft_launch_lines)
+
+    # Combine and write launch script
+    script_prefix = "\n".join(script_prefix_lines)
+    final_script = f"{script_prefix}\n\n{srun_args}\n\n{draft_launch_content}"
+
+    with open(launch_sh, "w") as f:
+        f.write(final_script)
+
+    # Make scripts executable
+    os.chmod(launch_sh, 0o755)
```

Ruff (0.14.14)

[error] `os.chmod` setting a permissive mask `0o755` on file or directory

(S103)